examples/jsm/utils/BufferGeometryUtils.js

```diff
@@ -1359,6 +1359,23 @@ function mergeBufferAttributes( attributes ) {
 
 }
 
+const BufferGeometryUtils = {
+
+	computeMikkTSpaceTangents,
+	mergeGeometries,
+	mergeBufferGeometries,
+	mergeAttributes,
+	mergeBufferAttributes,
+	interleaveAttributes,
+	estimateBytesUsed,
+	mergeVertices,
+	toTrianglesDrawMode,
+	computeMorphedAttributes,
+	mergeGroups,
+	toCreasedNormals,
+
+};
```

Comment:
This approach doesn't allow for tree-shaking inside the `BufferGeometryUtils.js` file, meaning that if you only use one function, you're gonna have to include the whole file.

You sure we can't use `export * as` yet?
This is not the main bundle, and CRA 4 users would have the old way of importing addons as an alternative as well.

Reply:
That's enough reason for me. FWIW namespaced exports are ES2020 or in browsers since early 2021 https://caniuse.com/mdn-javascript_statements_export_namespace.